userdocs/src/usage/autoscaling.md

```diff
@@ -44,7 +44,7 @@ nodeGroups:
       k8s.io/cluster-autoscaler/node-template/taint/feaster: "true:NoSchedule"
 ```
 
-For unmanaged noderoups, this is done by `eksctl` automatically if `propagateASGTags` is set to `true` like this:
+For unmanaged and managed noderoups, this is done by `eksctl` automatically if `propagateASGTags` is set to `true` like this:
```

Comment:
```suggestion
For unmanaged and managed nodegroups, this is done by `eksctl` automatically if `propagateASGTags` is set to `true` like this:
```

Reply:
fixed in 69390171dcd08ca0f96a6da8ebc8092a235e0cf8